tasks/factory_utils.py

```diff
@@ -0,0 +1,108 @@
+from abc import ABC, abstractmethod
```

Comment:
This module can be replaced by the one we added to CumulusCI, yeah?

Reply:
In general, I have been conservative about assuming cci will be deployed before this stuff is. After I merge, its "deployed" to anyone who does a git pull. Not admittedly, I guess, not many people will be running my code, but MetaCI will be...

I felt like it is a good habit to allow CCI a month or so to get new versions deployed before my code depends on those new versions.
 
Or at the very least I need MetaCI to have the new CCI.

It's easy to delete files later, so I decided to err on the side of backwards compatibility instead of risking support requests.

Reply:
MetaCI already has cumulusci 2.5.8. For people running CCI locally, you can update the first line of Cumulus' cumulusci.yml to say:

    minimum_cumulusci_version: 2.5.8

and then it will exit on startup with a message telling the user to upgrade.

Reply:
I decided not to be that strict because only a small subset of users (perhaps just me and MetaCI) care about these classes.